webapp/src/components/layout/Header.tsx

```diff
@@ -26,17 +26,17 @@ function LocaleSelectorContainer() {
 
 export default function Header({ logo, list, scrollOk }: Props) {
   return (
-    <header id="header" className={classNames({ scroll: scrollOk })}>
+    <header className={classNames('header', { scroll: scrollOk })}>
       {/* <LocaleSelectorContainer /> */}
-      <div id='left'>
+      <div className='header__left'>
         <img src={logo} alt={'logo'} className='logo'/>
         <ul>
           {list.map((element) => (
             <li key={element}>{element}</li>),
```

Comment:
element c'est trop vague, on pourrait avoir un nom plus descriptif ?

Reply:
navItem?

Reply:
ou navCategorie, navigationCategorie

Reply:
Je ne sais pas ce que c'est, donc trouve nom qui me permet de comprendre sans avoir à poser de question

Reply:
Je vois pas quoi mettre. J'ai cherché sur internet et j'ai trouve navigation categories ou navigation options.

Reply:
Oui ok enfaite ta props est mal nommé c'est pour ça.

list ça ne veux rien dire, en effet c'est des navItems.

```suggestion
          {navItems.map((navItem) => (
            <li key={navItem}>{navItem}</li>),
```